Dfe.Academies.Academisation.Data.UnitTest/ProjectAggregate/ProjectListGetDataQueryTests.cs

```diff
@@ -67,6 +68,45 @@ public async Task ProjectsExists_SearchProjectsByStatus__ReturnsProjects()
 		);
 	}
 
+	[Fact]
+	public async Task ProjectsExists_SearchProjectsV2ByStatus__ReturnsProjects()
+	{
+		// arrange
+		(_, Project project1) = CreateTestProject(null);
+		(_, Project project2) = CreateTestProject(null);
+		(_, Project project3) = CreateTestProject();
+
+		var projectDetails1 = _fixture.Build<ProjectDetails>().Create();
+		var projectDetails2 = _fixture.Build<ProjectDetails>().Create();
+
+		project1 = AddProjectDetailToProject(projectDetails1, DateTime.Now.AddDays(-1));
+		project2 = AddProjectDetailToProject(projectDetails2, DateTime.Now.AddDays(-2));
+
+		_context.Projects.Add(project1);
+		_context.Projects.Add(project2);
+		_context.Projects.Add(project3);
+
+		await _context.SaveChangesAsync();
+
+		// act
+		var searchStatus =
+			new string[] { project1.Details.ProjectStatus!.ToLower(), project2.Details.ProjectStatus!.ToLower() };
+		var projects = (await _subject.SearchProjectsV2(searchStatus, null, null, null, null, null, 1, 10)).Item1.ToList();
+
+		// assert
+		var firstProject = projects.FirstOrDefault();
```

Comment:
Would it be better to assert that the list contains both projects and a count of 2, rather than using first and last? Only so if the order of the list being returned changes at any point in the future for any reason this test would still work

Reply:
this was just a copy of an existing test and changed for v2, I will leave it for now and review all the tests in this file as we will probably remove the v1 search at some point